homework4/task4.py

```diff
@@ -0,0 +1,55 @@
+"""
+Write a function that takes a number N as an input and returns N FizzBuzz numbers*
+Write a doctest for that function.
+Definition of done:
+ - function is created
+ - function is properly formatted
+ - function has doctests
+ - doctests are run with pytest command
+You will learn:
+ - the most common test task for developers
+ - how to write doctests
+ - how to run doctests
+assert fizzbuzz(5) == ["1", "2", "fizz", "4", "buzz"]
+* https://en.wikipedia.org/wiki/Fizz_buzz
+** Энциклопедия профессора Фортрана page 14, 15, "Робот Фортран, чисть картошку!"
+"""
+from typing import List
+
+
+def fizzbuzz(n: int) -> List[str]:
+    """takes a number N as an input and returns N FizzBuzz numbers
+    >>> fizzbuzz(5)
+    ['1', '2', 'Fizz', '4', 'Buzz']
+    >>> fizzbuzz(0)
+    []
+    >>> fizzbuzz(-5)
+    []
+    >>> fizzbuzz(5.5)
+    Traceback (most recent call last):
+        ...
+    TypeError: 'float' object cannot be interpreted as an integer
+    >>> fizzbuzz(15)
+    ['1', '2', 'Fizz', '4', 'Buzz', 'Fizz', '7', '8', 'Fizz', 'Buzz', '11', 'Fizz', '13', '14', 'Fizz Buzz']
+
+    """
+    fizz_buzz = []
+    if not type(n) is int:
```

Comment:
`if type(n) is not int` is more readable

Reply:
fixed